### PR TITLE
1.5.0 news

### PR DESCRIPTION
**Pull request by roywei**
## Description ##
release notes for 1.5.0 release 
Do not merge until vote for 1.5.0 release is passed

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [ ] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [ ] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

**Comment by piyushghai** [comment]
Thanks for your contributions @roywei 
@mxnet-label-bot Add [Doc, pr-awaiting-review]

**Comment by Roshrini** [comment]
@aaronmarkham Can you please help review this PR?

File: NEWS.md [hunk 1]
```diff
@@ -17,6 +17,855 @@
...
+* Less cudaGet/SetDevice calls in Gluon execution (#13764)
+* Improve bulking in Gluon (#13890)
+* Increase perfomance of BulkAppend and BulkFlush (#14067)
+* Performance improvement in ToTensor GPU Kernel (#14099)
+* Performance improvement in Normalize GPU Kernel (#14139)
+* Bulked op segments to allow Variable nodes (#14200)
+* Performance improving for MKL-DNN Quantized FullyConnected (#14528)
+* speedup SequenceMask on GPU (#14445)
+* Dual stream cudnn Convolution backward() with MXNET_GPU_WORKER_NSTREAMS=2. (#14006)
+* Speedup _contrib_index_copy (#14359)
```
Comment (aaronmarkham) [inline NEWS.md]:
```suggestion
* Speedup `_contrib_index_copy` (#14359)
```
Add ticks to prevent preview from going awry.

File: NEWS.md [hunk 2]
```diff
@@ -17,6 +17,855 @@
...
+* Revert "Revert "[MXNET-43] Fix Jetson compilation" (#13665)" (#13672)
+* fix unpicklable transform_first on windows (#13686)
+* Fix NDArray ToDLPack Bug (#13698)
+* Fix the quantization script to support Python2 (#13700)
+* Update basic_layers.py (#13732)
+* [MXNET-1231] Allow not using Some in the Scala operators (#13619)
+* [MXNET-244] Work around likely compiler bug on nested inlines and temporary acces… (#13535)
+* Use curl to download sample data instead of wget. (#13761)
+* fix bipartite match memory corruption (#13727)
+* remove attributs clear on TRT nodes for GetOptimizedSymbol (#13703)
```
Comment (IvyBazan) [inline NEWS.md]: "attributes"

**Comment by roywei** [comment]
@aaronmarkham @IvyBazan Thanks for the review, I have addressed the comments.

**Comment by IvyBazan** [comment]
LGTM!

File: NEWS.md [hunk 3]
```diff
@@ -17,6 +17,855 @@
 
 MXNet Change Log
 ================
+## 1.5.0
```
Comment (aaronmarkham) [inline NEWS.md]: This TOC should work... but we might need to remove the old TOC for 1.4.1 (or update it) so there are no conflicts.
```suggestion
- [New Features](#new-features)
  * [Automatic Mixed Precision(experimental)](#automatic-mixed-precision-experimental-)
  * [MKL-DNN Reduced precision inference and RNN API support](#mkl-dnn-reduced-precision-inference-and-rnn-api-support)
  * [Dynamic Shape(experimental)](#dynamic-shape-experimental-)
  * [Large Tensor Support](#large-tensor-support)
  * [Dependency Update](#dependency-update)
  * [Gluon Fit API(experimental)](#gluon-fit-api-experimental-)
  * [New Operators](#new-operators)
- [Feature Improvements](#feature-improvements)
  * [Operators](#operators)
  * [MKLDNN](#mkldnn)
  * [ONNX](#onnx)
  * [TensorRT](#tensorrt)
  * [FP16 Support](#fp16-support)
  * [Deep Graph Library(DGL) support](#deep-graph-library-dgl--support)
  * [Horovod Integration](#horovod-integration)
  * [Dynamic Shape](#dynamic-shape)
  * [Backend Engine](#backend-engine)
  * [Large Tensor Support](#large-tensor-support-1)
  * [Quantization](#quantization)
  * [Profiler](#profiler)
  * [CoreML](#coreml)
- [Front End API](#front-end-api)
  * [Gluon](#gluon)
  * [Python](#python)
- [Language Bindings](#language-bindings)
  * [Scala](#scala)
  * [Java](#java)
  * [C++](#c--)
  * [Clojure](#clojure)
  * [Julia](#julia)
  * [Perl:](#perl-)
  * [R](#r)
- [Performance Improvements](#performance-improvements)
- [Example and Tutorials](#example-and-tutorials)
- [Website](#website)
- [Documentation](#documentation)
- [Build and Test](#build-and-test)
- [Bug-fixes](#bug-fixes)
- [License](#license)
- [Depreciations](#depreciations)
- [Known Issues](#known-issues)

## 1.5.0
```

**Comment by roywei** [comment]
@aaronmarkham Thanks for the TOC!! it seems working fine here: 
https://github.com/roywei/incubator-mxnet/blob/news/NEWS.md
what conflict are you talking about?

**Comment by aaronmarkham** [comment]
The old TOC that is down here:
https://github.com/apache/incubator-mxnet/pull/15137/files#diff-8312ad0561ef661716b48d09478362f3R1072
Might be better to always delete the old TOC and put a new one at the top. Of course it'll get super long...

**Comment by karan6181** [comment]
@IvyBazan Could you please review this PR? Thanks!

**Comment by IvyBazan** [comment]
LGTM!

**Comment by roywei** [comment]
@aaronmarkham 1.5 is released, this is good to go. We also need to update the website.

**Pull request merged**